select.go

```diff
@@ -86,11 +87,10 @@ func SelectNullStr(e SqlExecutor, query string, args ...interface{}) (sql.NullSt
 // SelectOne executes the given query (which should be a SELECT statement)
 // and binds the result to holder, which must be a pointer.
 //
-// If no row is found, an error (sql.ErrNoRows specifically) will be returned
+// # If no row is found, an error (sql.ErrNoRows specifically) will be returned
```

Comment:
Stray insert?
```suggestion
// If no row is found, an error (sql.ErrNoRows specifically) will be returned
```

Reply:
This one was weird. I kept editing it out and it would pop back in save. Eventually with trial and error I figured out gofmt's rule:

If there's a single line of text with blank lines above and below, and that line of text starts with a capital letter and it does not start with a period, it should be a heading.

So I added a period to satisfy gofmt.

Reply:
Where's my :scream: reactji when I need it